config/webhook.yaml

```diff
@@ -27,11 +27,15 @@ spec:
       role: webhook
   template:
     metadata:
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
       labels:
         app: webhook
         role: webhook
         discovery.knative.dev/release: devel
     spec:
+      serviceAccountName: controller
```

Comment:
same svc account as the controller? not using a different one?

Reply:
that is what we do elsewhere in knative https://github.com/knative/serving/blob/master/config/core/deployments/webhook.yaml#L47

Reply:
https://github.com/knative/eventing/blob/master/config/core/deployments/webhook.yaml#L43

Reply:
I'm ok with it though, just asking

Reply:
yeah true for eventing, but it might be overkill. We have had good success combining both the controller and webhook deployments and in that case it does not matter much.

Reply:
I can make an issue and track it, maybe we fix it long term.

Reply:
https://github.com/knative-sandbox/discovery/issues/37